src/services/users/index.js

```diff
@@ -152,11 +154,44 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (token) => {
+	try {
+		const decoded = jwt.verify(token, config('TOKEN_SECRET'))
+		const {email} = decoded
+		const result = await  db.query(
+			`SELECT * FROM user_invites
+		WHERE email = $1 
+		`,
+			[email]
+		)
+		const user = result.rows[0]
+		if (!user) {
+			throw customError(RefreshTokenIsInvalidError)
+		}
+		else if(user.status === "active"){
+			throw customError(InviteStatusActiveError)
```

Comment:
yea, we can merge the conditions like we were doing and use the invalid invite error

Reply:
yes please